src/Renderer/Shader/PointsVS.glsl

```diff
@@ -4,32 +4,38 @@ precision highp int;
 #include <logdepthbuf_pars_vertex>
 #define EPSILON 1e-6
 
-uniform float size;
-uniform float scale;
+attribute vec3 position;
 uniform mat4 projectionMatrix;
 uniform mat4 modelViewMatrix;
+
+uniform float size;
 uniform vec2 resolution;
-uniform bool pickingMode;
 
-attribute vec4 unique_id;
+uniform bool pickingMode;
+uniform bool useCustomColor;
```

Comment:
While you're improving this shader, could you define `useCustomColor` as a `float`, that would serve as:
```glsl
vColor = vec4(mix(color, customColor, useCustomColor), opacity);
```
?